### ISSUE TITLE
BUG: When NewDot is open on computer, getting notifications on mobile

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#11493]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Have NewDot Desktop open
2. Have someone send you message
3. Observe iOS notification even though Desktop is open and foreground

## Expected Result:
Not to receive iOS notification

## Actual Result:
iOS notification received

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Desktop App

**Version Number:** 1.2.11-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

<img src="https://user-images.githubusercontent.com/43996225/193378020-d9148124-a417-441e-80a7-fce22129d45a.PNG" width="40%" height="40%">

**Expensify/Expensify Issue URL:** 

**Issue reported by:** mattallen

**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1664405862076509

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by marcaaron** [comment]
Most likely `Internal` as the notifications for mobile are sent via the server.

**Comment by marcaaron** [comment]
I would guess [this PR](https://github.com/Expensify/Web-Expensify/pull/33495) might have caused the regression - but not sure. @techievivek are you able to follow up here and let us know if there's a link?

**Comment by techievivek** [comment]
@marcaaron Yeah, it could be related. I will have a look at it. Thanks

**Comment by mallenexpensify** [comment]
Dropped this in [the Slack thread](https://expensify.slack.com/archives/C01GTK53T8Q/p1664897756780179?thread_ts=1664405862.076509&cid=C01GTK53T8Q)
> Is anyone else is experiencing "When NewDot is open on computer, getting notifications on mobile" can you please comment in :thred: or on [the GH](https://github.com/Expensify/App/issues/11493), curious to see how widespread this is.

I know it was happening for Deeter too. 

I added this issue to[ the RCA Sheet](https://docs.google.com/spreadsheets/d/1u73RE8J1DdX3mDohV1RXgulF6_cke3N_a15Sk-UUdQo/edit#gid=0&range=F7) for "Reported bug GH issue link" @techievivek , if we find the root PR that caused this regression, can you update the sheet. If a C+ was assigned to review the PR, they should create the RCA using [the RCA template](https://docs.google.com/document/d/1UwXfA4ggO2h0XQJQlMEEQjT3zSphhSt12lpvuGSSFtw/edit?usp=sharing)

**Comment by techievivek** [comment]
@mallenexpensify This may be the result of backend changes that I have done in this [PR](https://github.com/Expensify/Web-Expensify/pull/33495) so we shouldn't bother any C+ for this.

**Comment by mallenexpensify** [comment]
K, thanks @techievivek , if it's because of changes you made, can you please fill out the RCA template? More details on steps are [here](https://docs.google.com/document/d/1BvohU05MTaHnjOD_vwJv_aqDAirv-ChkyRnKCAvOVyQ/edit#heading=h.8iom76ry1bp2)

**Comment by mallenexpensify** [comment]
Any update on getting this fixed or the PR reverted @techievivek ?   It's not necessarily a big priority, more annoying to be getting pinged multiple places

**Comment by techievivek** [comment]
Couldn't triage this because I am involved in the implementation of the `update concierge/newDot for AM` project but once I have some bandwidth, I will look into it.

**Comment by marcaaron** [comment]
@techievivek as this is a Daily I would recommend providing whatever details you have and removing your assignment so someone else can look into it.

**Comment by techievivek** [comment]
Not at the moment to be honest because even the linked [PR](https://github.com/Expensify/Web-Expensify/pull/33495) was in the open state for a very long but since the code changes in the PR are very less one will be able to figure out if the linked PR is the cause of regression. If no one picks this up by the weekend I can look into it.

**Comment by mallenexpensify** [comment]
Posted in #engineering to get more 👀 https://expensify.slack.com/archives/C03SSAQ7P/p1665449068132379

**Comment by Justicea83** [comment]
I don't know if this is related. But when a chat is active and open, do I still need to receive notifications?

https://user-images.githubusercontent.com/26106822/195350426-d9436077-a1c8-4491-903b-6033301e00ab.mp4

<img width="634" alt="Screenshot 2022-10-12 at 1 06 15 PM" src="https://user-images.githubusercontent.com/26106822/195350604-73901ca1-7006-43e4-9a11-e810677b4bf2.png">

**Comment by Justicea83** [comment]
There's a [WN](https://expensify.slack.com/archives/C01GTK53T8Q/p1665580878349749?thread_ts=1665580599.621459&cid=C01GTK53T8Q) on notifications rework, as there're a wholesale bugs on nofications. So @techievivek you can close this out unless you already have a solution.

**Comment by mallenexpensify** [comment]
@Julesssss , you're working on Notifications, right?  Any thoughts on why this is happening? It'd def a regression. 
Is there a tracking issue with a list of Notification issues?  
Let's not close cuz we want to be able to confirm we actually fixed the bug before closing

**Comment by marcaaron** [comment]
I agree this is a very annoying regression and should be prioritized highly.

**Comment by Julesssss** [comment]
THis will definitely be covered by the upcoming doc, but given this seems to be a regression perhaps there also a revert/fix that can be applied in the meantime?

**Comment by Julesssss** [comment]
@mallenexpensify here' the tracking issue for collecting Notification issues: https://github.com/Expensify/Expensify/issues/219254

**Comment by mallenexpensify** [comment]
Do we have anything written down for how to track down the offending PR for regressions? For internal and/or contributors?  (I have zero clue)

**Comment by mallenexpensify** [comment]
Dropped in #engineering-chat (thought it was #engineering) to get 👀 
https://expensify.slack.com/archives/C03TQ48KC/p1666027005682349

**Comment by techievivek** [comment]
@roryabraham had a look at it during our sync and my [Web-E PR](https://github.com/Expensify/Web-Expensify/pull/33495) indeed caused this regression. Here is what I changed that caused the regression.

```php
- 'alert' => $isSilentPush ? '' : $message
+ 'alert' => $message
```

I will put a PR today fixing it today.

**Comment by mallenexpensify** [comment]
Thanks @techievivek , I added a [comment on the RCA sheet](https://docs.google.com/spreadsheets/d/1u73RE8J1DdX3mDohV1RXgulF6_cke3N_a15Sk-UUdQo/edit#gid=0&range=C7) asking you to fill out the [RCA template](https://docs.google.com/document/d/1UwXfA4ggO2h0XQJQlMEEQjT3zSphhSt12lpvuGSSFtw/edit?usp=sharing) once this is fixed.

**Comment by techievivek** [comment]
The PR is up but based on the [discussion](https://expensify.slack.com/archives/C01GTK53T8Q/p1666197911647179?thread_ts=1666109949.817589&cid=C01GTK53T8Q) here I am not sure if we are going to fix this.

**Comment by Julesssss** [comment]
Thanks @techievivek, yep I'd say we should just close this. We decided to continue showing notifications on all platforms for the moment.

**Comment by mallenexpensify** [comment]
> We decided to continue showing notifications on all platforms for the moment.

What's the reasoning or link for this. I find it to be 'low quality' to be in Desktop and have my phone 'light up' all the time with messages.  Is there a link to a discussion?

**Comment by Julesssss** [comment]
Hey @mallenexpensify, here's the[ pre-design thread.](https://expensify.slack.com/archives/C01GTK53T8Q/p1666109949817589)

**Comment by mallenexpensify** [comment]
Thanks @Julesssss , sorry I missed that.   Quite the 🐰🕳, couldn't help but comment https://expensify.slack.com/archives/C01GTK53T8Q/p1666626222041809?thread_ts=1666109949.817589&cid=C01GTK53T8Q
